404.md

```diff
@@ -0,0 +1,3 @@
+This usually means that you've typed or followed a bad URL. Occasionally, this can happen if the page has been moved to a new URL (though we try to avoid doing that).
```

Comment:
I think the file for this repo should be inside `docs/` as this is where its markdown files are stored